app/test_main.py

```diff
@@ -1,3 +1,95 @@
+import pytest
+from typing import Any
 from app.main import get_human_age
```

Comment:
fix imports. You should put a blank line between each group of the imports.  You can find info here https://peps.python.org/pep-0008/#imports
![Screenshot from 2023-12-06 19-07-47](https://github.com/mate-academy/py-cat-and-dog-years/assets/122517203/5784fb39-a80d-45a1-a3b3-7c76784e5936)